### DISCUSSION TITLE
TailwindCSS 2.2.2 (JIT mode) does not apply style in Nextjs 11

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#4731] [Help]
I tried to update both Next.js from 10 to 11 and TailwindCSS from 2.1.2 to 2.2.2 (JIT Mode).
After I upgrade, all styles is not applied. Sometime, some Tailwind Classes are applied and some are not.
I tried to install a scratch next-app with lastest Next.js and lastest TailwindCSS as well, and style is not applied. Repo example here: https://github.com/huuphongnguyen/nextjs11tailwind22
Does I miss something or mess in something in setup?
Thank you so much.

**Comment by simonswiss** [comment]
Hey @huuphongnguyen 👋

Do you have examples of what classes are not working for you? I cloned your repo and started adding classes (including arbitrary values) and everything worked.

If you tell me which ones don't work, that will help troubleshoot 👍
